src/content/tools/pub/cmd/pub-upgrade.md

```diff
@@ -56,12 +56,21 @@ $ dart pub upgrade test args
 Dependencies upgraded!
 ```
 
-Upgrading a dependency upgrades its transitive dependencies to their latest
-versions as well. Usually, no other dependencies are updated; they stay at the
+Usually, no other dependencies are updated; they stay at the
```

Comment:
Can we say "upgraded" here just to keep using the same language? Not sure if this only use of "update" is intentional or not:
```suggestion
Usually, no other dependencies are upgraded; they stay at the
```